Smartway.UiComponent/Inputs/NumericDateEntry.xaml.cs

```diff
@@ -65,6 +65,21 @@ public ICommand ErrorCommand
             set => SetValue(ErrorCommandProperty, value);
         }
 
+        private CultureInfo _culture;
```

Comment:
Pour résumer, l'objectif c'est que dans l'appli on utilise ce composant systématiquement avec une culture à "calendrier grégorien" ?

Reply:
Hmm, je ne comprends pas la nuance. On est bien d'accord que l'objectif c'est que dans l'app, on n'utilise plus ce composant que dans sa version "avec spécification d'une culture" ?

Reply:
Ah, oui, j'avais mal lu la question. J'avais compris "on utilise systématiquement ce composant quand on veut afficher une date" ><

Oui, on utilisera systématiquement avec une culture spécifiée

Reply:
Aaah ok, merci ;)

Reply:
Par contre je me rends compte que ce n'est pas exactement ce que j'avais compris ce matin. Je pensais que dans le ToString on pourrait préciser un type de calendrier, pas une culture. Du coup par exemple on passera quoi comme "culture" dans l'app pour in fine avoir un calendrier grégorien ?

Reply:
> Par contre je me rends compte que ce n'est pas exactement ce que j'avais compris ce matin. Je pensais que dans le ToString on pourrait préciser un type de calendrier, pas une culture

Je pensais aussi ça, sauf que pour formatter un datetime, il n'attend pas juste un calendrier mais un IFormatProvider (que CultureInfo implémente) car il y a d'autres paramètres (comme l'ordre JOUR / MOIS / ANNEE) de certains affichages.

Ce que je suis en train de faire côté app xamarin, c'est que quand c'est la culture thai, je fournit une autre culture au toString (pour l'instant je suis partit sur la français, mais c'est le temps de mes tests. Il faudra qu eje trouve la plus adaptée) et pour toutes les autres cultures, je fournis la culture du device. Comme ça on impacte uniquement le thai

Reply:
Ah ! Bon ça me rassure j'étais pas à côté de la plaque x)
C'est dommage ça nous fait faire vraiment un gros hack mais tant pis... Si déjà on n'envoie une culture que dans les cas ou c'est du thaï c'est un moindre mal